app/controllers/jobs_controller.rb

```diff
@@ -22,18 +45,36 @@ def create
 
   def show
     @job = Job.find(params[:id])
+    @company = @job.company
+    @comments = @job.comments.reverse
+    @comment = Comment.new
+    @comment.job_id = @job.id
```

Comment:
no need to do this here because we aren't saving this comment here. we just need to set this in the create method